homeassistant/components/luxor/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
```

Comment:
The config step strings are missing here

Reply:
I added the Host translation and added a title.